### ISSUE TITLE
Error: Cannot open services.conf.  The System cannot find the path specified

### ISSUE THREAD

**Issue by tinqlik** [eapowertools/QlikSenseQMCUtility#86]
Hi, 
I am getting an issue with the Dispatcher step of the installation
[QMCUtilities Install Error.docx](https://github.com/eapowertools/QlikSenseQMCUtility/files/1092709/QMCUtilities.Install.Error.docx)

**Comment by jparis** [comment]
Now... That's a strange error. @goldbergjeffrey Seems like something is up with the installer?

Also @tinqlik where is your Qlik Sense installation? All the services are there, but the files are not in `C:\Program Files\Qlik\Sense`?

**Comment by tinqlik** [comment]
Yes I think the culprit is because I did not use the default folder when installing Qlik Sense Enterprise on my local laptop, so we should probably put a disclaimer that the tool will only install if you have used the default install location (unless we can make the utility installer flexible and have the user point to the Enterprise install root folder).
In addition, I noticed there is not a graceful way to exit the utility installer (Cancel button is missing).  I am going to try and uninstall my Enterprise server and then reinstall using the default locations.

**Comment by tinqlik** [comment]
Re-installing with the Default Qlik Sense Enterprise path allowed me to complete the utility installation successfully.

**Comment by MaximilianNK** [comment]
Hi. I have totally the same problem. but reinstalling production server just to make utilites working looks little bit strange for me.  
Is there is any way available to install QMC Utilities if QlikSense installed on non-default path?

**Comment by jparis** [comment]
Leaving open as issue in the installer then. Installer fails if not using default Qlik Sense install path. You can manually put the files in place, or fix the installer bug as an interm solution. What plugin are you looking to use?

**Comment by tinqlik** [comment]
Hey Jesse,
I forget, I think I was just installing the whole thing just to get acquainted the PowerTools when I first started.  I ended up going back and installing QS using the default path so it would work.

Regards,


Thomas Lin
Partner Pre-Sales Enablement Manager

Mobile: +1 469-964-2136
Skype: thomaslin_qlik
Email:  Thomas.Lin@qlik.com

Qlik


qlik.com<http://www.qlik.com/>

The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any review, retransmission, dissemination or other use of, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you received this in error, please contact the sender and delete the material from any computer.
